bin/i18n/sync-codeorg-out.rb

```diff
@@ -36,6 +36,24 @@ def rename_from_crowdin_name_to_locale
   end
 end
 
+def find_malformed_links_images(locale, file_path)
+  return unless File.exist?(file_path)
+  is_json = File.extname(file_path) == '.json'
+  data =
+    if is_json
+      file = File.open(file_path, 'r')
+      JSON.load(file)
+    else
+      YAML.load_file(file_path)
+    end
+
+  return unless data&.values&.first&.length
+  crowdin_file_path_match = file_path.match(/i18n\/locales\/[a-z]+\-[A-Z]+\/(.+)/)
```

Comment:
it's not clear to me why this is necessary. Can we not just use the `file_path`?

Reply:
I was trying to get rid of the `i18n/locales/` part of the filepath but now that I think about it, it's probably too complex for what it's worth. I'll remove this.